Agocs/template/buggy_function.py

```diff
@@ -20,8 +20,10 @@ def angle_to_sexigesimal(angle_in_degrees, decimals=3):
     if math.floor(decimals) != decimals:
         raise OSError('decimals should be an integer!')
 
-    hours_num = angle_in_degrees*24/180
+    hours_num = angle_in_degrees*24./360.
     hours = math.floor(hours_num)
```

Comment:
I think there is bug here that needs fixing, since
floor(2.3) -> 2 but
floor(-2.3) -> -3
Could you double check that this is what is intended?

Reply:
I think this depends on whether we want to allow the user to input negative values for the angle; you're right that the code should definitely not fail directly and a warning/error message is necessary at least